tsl/src/reorder.c

```diff
@@ -419,17 +419,17 @@ chunk_get_reorder_index(Hypertable *ht, Chunk *chunk, Oid index_relid, ChunkInde
 /* The following functions are based on their equivalents in postgres's cluster.c */
 
 /*
- * timescale_reorder_rel
+ * reorder_rel
  *
  * This clusters the table by creating a new, clustered table and
  * swapping the relfilenodes of the new table and the old table, so
  * the OID of the original table is preserved.
  *
  * Indexes are rebuilt in the same manner.
  */
-void
-timescale_reorder_rel(Oid tableOid, Oid indexOid, bool verbose, Oid wait_id,
-					  Oid destination_tablespace, Oid index_tablespace)
+static void
+reorder_rel(Oid tableOid, Oid indexOid, bool verbose, Oid wait_id, Oid destination_tablespace,
```

Comment:
I find myself adding prefixes instead, when there's a postgres function with the same name. I always confuse them in backtraces, switch to the wrong one in "go to definition", change the wrong one and don't understand why it doesn't work, it's really annoying :)

Reply:
Hmm i'm fine with either here but timescale_ is certainly the wrong prefix.

Reply:
Is it for public exports? IIRC I added `ts_` to some stuff from /import

Reply:
we use ts_ for exported stuff, we don't really have rules for local prefixes but timescale is the company name so if we wanted to use a long prefix it should have been timescaledb_